jurity/recommenders/rank_estimation.py

```diff
@@ -0,0 +1,47 @@
+from typing import NamedTuple
+
+import numpy as np
+import pandas as pd
+
+from jurity.utils import Constants, get_sorted_clicks
+
+
+class RankEstimation:
```

Comment:
Thinking out loud. Instead of passing this class as an argument to each of the ranking reco metrics, would it work to have this class inherit from `_BaseRecommenders` and then to have each of the ranking reco metrics inherit from this class? (could probably call in something like `_BaseRankingRecommenders`)

Could even consider implementing the `get_correction()` method directly in `_BaseRecommenders` and only executing if necessary arguments (n_items, n_samples?) are specified?

Reply:
I like that idea! But for the API design are we ok with having these two arguments be supplied separately even though both are required for this functionality?

Reply:
We could have a required check for both arguments in the beginning of that function?